Power Up Proto/src/org/usfirst/frc/team5493/robot/subsystems/Solenoid.java

```diff
@@ -11,9 +11,9 @@
  */
 public class Solenoid extends Subsystem {
 	
```

Comment:
Are we really only going to have one `Solenoid`? Is this better named `ShiftSolenoid `or something like that?